File: pytools/examples/golang/src/server/proto/ack_role_login_ok_no_role.go
package proto

import (
	"packet"
)

type AckRoleLoginOkNoRole struct {
}

func (this *AckRoleLoginOkNoRole) Encode() []byte {
	pack := packet.NewWriteBuff(64)


	return pack.Encode(P_ACK_ROLE_LOGIN_OK_NO_ROLE)
}

File: pytools/data/data.code.cpp.server/AckRoleLoginOkNoRole.cpp
#ifndef _ACK_ROLE_LOGIN_OK_NO_ROLE_
#define _ACK_ROLE_LOGIN_OK_NO_ROLE_

#include <list>

#include <pb_type.hpp>
#include <Packet.hpp>

#include <Msg.hpp>



class AckRoleLoginOkNoRole
{
private:


public:
	Packet Encode()
	{
		Packet packet;
		packet.Encode(Msg::P_ACK_ROLE_LOGIN_OK_NO_ROLE);
		return packet;
	}


};

#endif
